### PR TITLE
Add elastic upgrade instructions hard update

### LINKED ISSUE #2268: Add intructions to add again Elastic's repository when upgrading  from 6.x to 6.8
Hello team! This issue aims to add instructions to add in case of having disabled Elasric's repository when upgrading from 6.x to 6.8.

Regards,

David

### PR DESCRIPTION
**Pull request by davidfmiranda**
Hello team!

This pull request closes #2268 

## Description

I have added instructions to enable the repository in case it had been disabled in hard and minor upgrade section: 

enable-repo

Regards,

David

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar.

### CODE REVIEW

File: source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 .. _elastic_server_hard_upgrade:
 
-Upgrading Elastic Stack from 6.x to 6.8
+Upgrading Elastic Stack from 6.x to 6.x
```
Comment (alberpilot) [inline source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst]: This must be 6.8. Elastic, in theory, will not release a 6.9, the 6.8 will be permanent and all changes on this version will be on patches.

File: source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst [hunk 2]
```diff
@@ -49,13 +70,13 @@ Upgrade Elasticsearch
 
     .. code-block:: console
 
-      # yum install elasticsearch-6.8.6
+      # yum install elasticsearch-6.x.6
```
Comment (alberpilot) [inline source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst]: This `x` doesn't work. This branch must be updated with the 3.11 latest changes `6.8.7`.

File: source/upgrade-guide/upgrading-elastic-stack/elastic_server_minor_upgrade.rst [hunk 3]
```diff
@@ -15,47 +15,26 @@ Prepare the Elastic Stack
       # systemctl stop filebeat
       # systemctl stop kibana
 
-#. Add the new repository for Elastic Stack 7.x:
+#. In case of having disabled the repository for Elastic Stack 7.x it can be enabled using:
```
Comment (alberpilot) [inline source/upgrade-guide/upgrading-elastic-stack/elastic_server_minor_upgrade.rst]: 7 or 6?

File: source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst (hunk 1 above)
Comment (davidfmiranda) [reply source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst]: Done!

File: source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst (hunk 2 above)
Comment (davidfmiranda) [reply source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst]: Done!

File: source/upgrade-guide/upgrading-elastic-stack/elastic_server_minor_upgrade.rst (hunk 3 above)
Comment (davidfmiranda) [reply source/upgrade-guide/upgrading-elastic-stack/elastic_server_minor_upgrade.rst]: It has to be 7, because it is a minor upgrade from 7.y to 7.x

**Pull request merged**